components/Feature/VulnerabilitiesGrid/ResourceCard/index.js

```diff
@@ -0,0 +1,34 @@
+import css from './index.module.scss';
+
+const ResourceCard = ({ name, description, websites, address }) => (
+  <div className={`govuk-details__text ${css.resource}`}>
+    <h3>{name}</h3>
+    {websites && websites.length > 0 && (
+      <>
+        <ul className={css.websites}>
+          {websites.map(website => (
+            <li key={website}>
+              <a href={website} target="_blank" rel="noopener noreferrer">
+                {website}
+              </a>
+            </li>
+          ))}
+        </ul>
+      </>
+    )}
+    <details className="govuk-details" data-module="govuk-details">
+      <summary className="">View more</summary>
+      <div className={css.details}>
+        <p>{description}</p>
+        {address && (
+          <>
+            <h4>Address</h4>
+            <p>{address}</p>
+          </>
+        )}
+      </div>
+    </details>
```

Comment:
The styling on this would mean quite a few overrides to the existing `<Details />` component as it exists in this project, hence the vanilla `<details>` component here.

Reply:
I don't think we actually use that <Details /> atm but all good